script/1_0_2current_en_script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -0,0 +1,130 @@
+<Page16>
+...I see. So that's how our relationship looks like in her eyes.
+<Page17>
+"Hold on Akiha. First things first, we don't have that kind of relationship.
+All I'm doing is helping her find something, and once that's over, I have no other reason to see her again."
+<Page18>
+"...That's odd, considering how awfully close you looked, but I guess you would say that it was just my imagination.
+Are you telling me that this is simply a temporary, give-and-take relationship?"
+<Page19>
+Akiha's gaze is cold.
+If I nod out of confirmation, this conversation could be settled peacefully.
+...But I feel strangely reluctant in affirming that statement.
+Although we both have something to gain by working together, I find it hard to say that is all there is to it.
+<Page20>
+"Well that is... we're not exactly close, but it's not like I'm helping her purely out of give-and-take either."
+"Of course. After all, I've never seen you in a panic like that before. To see Nii-san who would never take anything seriously to reveal his emotions like that, naturally I would question what on Earth would have brought that on."
+<Page21>
+"I didn't panic that hard, and I'm not that cold either. I was just surprised because I didn't expect her to visit our house."
+"Really now? Even though that woman remained totally calm as she waited for you for that long?"
+<Page22>
+Akiha gives me a sidelong glance like she had something to say.
+It seems that she thoroughly believes that Arcueid and I are going out.
+<Page23>
+"Hey Akiha. Could you stop with the misunderstandings?
+Besides, the only reason we know each other was because―――"
+<Page24>
+I stop myself from saying that it was because we killed him.
+...It's not like she would believe me even if I did tell her the truth. And besides, there's no way I could let her know about that sort of thing.
+That being said, I can't think of any other better explanation, so I had no choice but to evade the question and mumble the same words as last night.
+<Page25>
+"If you cannot answer the question, then perhaps I should change it.
+You've been constantly referring to her as 'that woman' for a while now, but what exactly is her name?"
+"Her name is... Arcueid."
+<Page26>
+"Is that so? Then that means you two didn't know each other from school. There are no exchange students in your school after all."
+<Page27>
+"*agh*"
+It's just as Akiha says, there isn't a single exchange student from abroad in our school... But even so, why does Akiha know about that?
+<Page28>
+"And so, how exactly did you two come to know each other?"
+"W-We met each other in the city. By accident."
+<Page29>
+"By accident, you say. So then Arcueid was the one that called out to you, correct?"
+"Ah―――well, actually it was me who called out to her..."
+In reality, it was me killing her then becoming chased afterwards rather than me calling her out. In any case, I was the one who triggered the situation.
```

Comment:
> 実際は声をかけたのではなく殺害し、その後に追いかけられた。ともあれ、きっかけを作ったのは俺の方だ。
The truth is that there was no calling out to anybody, only murder. After that there was the chase. Anyhow, I was the one who created the situation.